app_dart/bin/server.dart

```diff
@@ -139,6 +139,24 @@ Future<void> main() async {
       ///
       /// Response: Status 200 OK
       '/api/update-task-status': UpdateTaskStatus(config, swarmingAuthProvider),
+
+      /// Apply task updates via PubSub messages.
+      ///
+      /// This API updates task status in datastore which have been
+      /// communicated via PubSub messages.
+      ///
+      /// POST: /api/task-status-subscription
+      ///
+      /// Parameters:
+      ///   CommitBranch: (string in body). Branch of commit.
+      ///   CommitSha: (string in body). Sha of commit.
+      ///   BuilderName: (string in body). Name of the luci builder.
+      ///   NewStatus: (string in body) required. Status of the task.
+      ///   ResultData: (string in body) optional. Benchmark data.
+      ///   BenchmarkScoreKeys: (string in body) optional. Benchmark data.
+      ///
+      /// Response: Status 200 OK
+      '/api/task-status-subscription': TaskStatusSubscription(cache, config, authProvider),
```

Comment:
Namespace this into `/api/results/task-status-subscription` to make it easier to fork into a dedicated service

Reply:
done